### DISCUSSION TITLE
Multiple users on the same Windows device without changing Windows accounts.

### DISCUSSION THREAD

**Discussion by DujeVatavuk** [passwordless-lib/fido2-net-lib#516] [Q&A]
Hi, 

I am working on an blazor server application for a restaurant and we have a windows computer on which all of the deliverers and other staff can manage orders. In order to improve the workflow and we wanted to implement biometric authentication/identification via fingerprint instead of writing a password, or even better if that fingerprint can be used without the username. 

I tried to implement couple of demos that I saw in your documentation and I have also tried your already built webpages but I never found a way I could register a new person with their username/email data and their biometrics on a windows device on which I was already logged in. 

Is there an option that under the same windows user we could run the application and do a biometry log in (with or without the username) for multiple people? (it would be ok if we had to created a windows account/user for every person in the staff with registered biometrics on that device)

**Comment by aseigler** [comment]
This is more of a client thing than a server thing.  You can definitely create multiple passkeys under the same local Windows user accounts for multiple people, that is much more obvious in newer builds of Windows 11 than it was before.  The issue is I think is going to be the biometrics, I am not sure what the limitations are there out of the box.
